### DISCUSSION TITLE
editable order confirmation text

### DISCUSSION THREAD

**Discussion by FlohGro-dev** [pretix/pretix#1980] [Q&A] [answered]
Hi, 
I spend a lot of time and did not found anything.
Is it not possible to change the text which is displayed after a successful order (no matter if the ticket needs an approval or not)?
We want to add some minor information like "please check your spam folder" and so on to this page.

Please see the screenshot (sorry the language is set to German but I think this should clarify which text I mean.
<img width="723" alt="Bildschirmfoto 2021-02-25 um 20 40 31" src="https://user-images.githubusercontent.com/13785667/109208015-170ff700-77aa-11eb-806f-dae78ad1f788.png">

**Comment by raphaelm** [comment, marked as answer]
Sorry, no, it is not possible to change pretix's default texts.

**Reply by FlohGro-dev** [reply]
ok, this would be really great if it would be editable.
Just an optional text and if it is not set, use the pretix default one.
Otherwise I need to e.g. use a plugin like [Pages](https://marketplace.pretix.eu/products/pages/) to display this informations during the ordering process.

**Reply by raphaelm** [reply]
I understand why you'd want this, but I'm worried about adding too many configuration options, which makes it hard to update pretix in the future, and at the same time makes it harder for new users to find they things they're looking for.

**Reply by FlohGro-dev** [reply]
I don't know if this will make the configuration more complex. This could be an easy addition with a description in the Settings-General-Texts section.

**Reply by raphaelm** [reply]
It wouldn't bee that easy, since the text isn't a static text, but composed of 7 different texts depending on situation (free ticket, already paid, payment pending, approval required…)

Nevertheless, 8af91b691d0d4ce9c5ed8af25c73ad1c36460af2 now adds the option to put in such a text that will be shown *in addition* to the standard text.

**Reply by FlohGro-dev** [reply]
thank you very much!
